Verbiage changes

Jarli01 opened the pull request:
More clear statements of warranty, particularly for source installations and support clauses and grammar.

### Check list

> Check items when done or if not relevant

- [ ] PR reference the relevant issue (e.g. `Fixes #007`)
- [X] if UI changes, a screenshot has been added to the PR
- [ ] `CHANGELOG.unreleased.md`:
   - enhancement/bug fix entry added
   - list of packages to release updated (`${name} v${new version}`)
- [ ] documentation updated
- [X] **I have tested added/updated features** (and impacted code)

### Process

1. create a PR as soon as possible
1. mark it as `WiP:` (Work in Progress) if not ready to be merged
1. when you want a review, add a reviewer
1. if necessary, update your PR, and re- add a reviewer

Jarli01 committed:
Verbiage changes

More clear statements of warranty, particularly for source installations.

olivierlambert commented on packages/xo-web/src/common/intl/messages.js:
```diff
@@ -1789,9 +1789,9 @@ const messages = {
-  noProSupport: 'No pro support provided!',
-  noProductionUse: 'Use in production at your own risks',
-  downloadXoaFromWebsite: 'You can download our turnkey appliance at {website}',
+  noProSupport: 'Limited Support Provided!',
```
No, there is no professional support at all (as the translate key indicates)

Jarli01 replied:
This is for the source installation, yes? So as a reasonable change to the verbiage since "No Pro Support" really doesn't make sense as anyone can take the source code and sell support for it. 

Changing the verbiage here makes the most sense, as Support is limited.

Jarli01 replied:
And "Limited to You build it yourself, support it yourself or purchase the support by going to our website and getting XOA and signing up for a plan".

olivierlambert replied:
If one want to sell XO support from the sources, it will be done via a fork (and changing various HTML pages to point to their own website). Then, it would be logical to also change this translation. That's why, for something freshly cloned on GitHub, "No pro support provided" seems good to me. The original meaning is "You won't have company-backed support (aka pro support) on this code as-is". That's why I think it's clear enough.

Regarding your alternative: it's a lot longer (info density far lower). It will be probably less informative in the end.

What to you want to express here on changing this?

Jarli01 replied:
So if I built from source, followed your exact guide, and it didn't work, you'd tell me to pound sand?

The "Limited Support Provided" is very clear in what is provided, and this change can be made down stream if you'd so prefer but seems counter intuitive to make the change down stream. As being in the source makes way more sense. 

As anyone who goes to the forums and asks for help can get it, some times even from the dev team. It's also why you ask on every topic are you using "XOA or from the source?".

olivierlambert replied:
> So if I built from source, followed your exact guide, and it didn't work, you'd tell me to pound sand?

If you build from sources and you have issues, you can't have **pro support** (pro support != community support), so yes, that's exactly what I meant! I thought it was clear enough with 3 words "No pro support".

So again: there is NO **pro** support if you build it from GitHub. Again, pro support = support backed by a company, not by the user community.

Is it more clear?

Jarli01 replied:
> So again: there is NO **pro** support if you build it from GitHub. Again, pro support = support backed by a company, not by the user community.

This makes absolutely zero sense. 

Either accept the changes or decline them, with the way you're acting I'm amazed that this project is progressing at all. As the verbiage and language in here is very much Oh you're not paying me, so piss off. 

Which is fine but there is a better way do to it. 

Also how do you think RedHat and CentOS / Fedora operate?

olivierlambert replied:
As a company, we absolutely can't provide support on something installed from the sources. It's the same for RedHat: they won't provide pro support I you decided to install Linux from scratch and build everything yourself from various git repositories.

RH is providing tested RPM packages and a distro. XOA is like that, "the final product", tested and validated, where you can reasonably provide support because there is a controlled build environment. A lot other projects are like this, eg Linstor from Linbit, providing everything on GitHub without any pro support, but having RPMs/deb validated packages to support them.

So I repeat again: if you are building from the GH repo, you can't have pro support, that's the way it is and it's pretty common. In Open Source, value is often on support and turnkey/QA (aka packaged/validated) solution.

What is **pro** support? When people with pro support ask for something, we'll investigate on their XOA remotely.

What's not pro support? We can fix issues reported on GitHub but we don't fix it on people setup directly, and we won't be able to investigate deeply.

This project is build by Vates at 99% and we chose -from the start, because we love Open Source- to give it to the community for free on GitHub. Everybody can contribute (all contributions are **very welcome**).

To recap, there's no support from XO cloned on Github. If you want to add a extra link "Community forum" somewhere near, that's fine for me.

Jarli01 replied:
Look at my GH that you were recently paged on. and do you mean like this?

![IPrMbkhW2Z](https://user-images.githubusercontent.com/14837699/57724060-f66bc100-7657-11e9-877e-f49813e4541b.png)

olivierlambert replied:
When I said "add an extra link "Community forum" somewhere near", I meant to add an extra link "Community forum" somewhere near (the line after probably), no replacing the whole page.

Also, I'd like to remind that "Trolling/insulting" is not being helpful at all and explicitly described as unacceptable behavior in our [code of conduct](https://github.com/vatesfr/xen-orchestra/blob/master/CODE_OF_CONDUCT.md).

Jarli01 replied:
Considering you don't offer professional support for the Source installation, changing that line would make the most sense, since and I quote "NO **pro** support if you build it from GitHub"

Which means, anyone who installs from source down stream using a script or otherwise should have nowhere to go for support. 

You can't have it both ways, pick one. Either drive people to your GH asking for assistance with the source installation or be accepting of a possible downstream change to the source post install that drives people to the appropriate channel for support before the VatesFR GH page.

Jarli01 replied:
Also nothing in what I've posted on this topic is trolling in nature or being insulting. You've refused to accept the changes, so either close the PR or accept them. 

As the documentation shows currently I find it unacceptable and may just make this change downstream and not include you going forward.

olivierlambert replied:
Not in this topic, on your repo. Calling someone a prick is definitely entering in this code of conduct, and not helpful to discuss about modifications. This is not how to collaborate and discuss upstream work.

I'm OK to merge a PR with the suggested changes we discussed, otherwise, feel free to close it.

Jarli01 committed:
Update messages.js

That's the best compromise for the verbiage here that makes sense and isn't off putting.

olivierlambert commented:
Looks good to me, thanks :+1:
